x-pack/platform/plugins/private/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/reindex/flyout/checklist_step.tsx

```diff
@@ -197,6 +198,26 @@ export const ChecklistFlyoutStep: React.FunctionComponent<{
               }}
             />
           </p>
+          {frozen && (
+            <p>
+              <FormattedMessage
+                id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.reindexFrozenIndex"
+                defaultMessage="Frozen indices will no longer be supported after upgrade, so this index be deleted as part the reindex operation. {docsLink}"
```

Comment:
I think we should get someone from Kibana docs to review this copy. Perhaps @florent-leborgne

Reply:
If we put this in a callout we can add a title like: `This index is frozen`